Functions/xASL_vis_CreateVisualFig.m

```diff
@@ -343,8 +343,12 @@
 % requested. If DirOut is provided, this image is also saved as jpg file
 if numel(DirOut)>1 && numel(ImOut)>1 % when DirOut or ImOut==NaN, skip this
     xASL_adm_CreateDir(DirOut);
+	FileName = NamePrefix;

```

Comment:
How does this work when `pWM` is overlaid over `qCBF`, this would become `qCBF_pWM_sub-001_ASL_1.jpg` right? That sounds good, though I wonder how this works with other functions that may expect the previous naming convention, did we test this?

Reply:
it gives pWM_sub_001_qCBF_sub_001_ASL_1.jpg - so there is some redundancy.

We can try to be a bit more smart to remove this redundancy, but I don't see it as a disadvantage to have full names of both. Because previously we have only the name of the last file and that was not really helpful sometimes.

Previous naming convention? We only save those to disk, we don't reload them. I don't see a conflict there. YOu can test and see if you like it...

Reply:
No the prefix dealth with this right? The prefix would be `pWM` in this case. I like your approach, which is smarter, but you cannot do this easily without testing... If 1.7.0 needs to be out soon I would move this to 1.8.0

Reply:
The prefix dealt with it, but it didn't have the patient session. So I kept the prefix and added the additional names. 

So it works, but it is a but superfluous. But that's not an issue, because these files are written and nowhere read again?! Right, so it can't cause any conflict, or can it?

I actually really want it in 1.7.0 for Wibeke. They have multi-session ASL, and we can't check the registration of each session, because it always overwrites it with the latest session. So rather annoying bug

Reply:
Consensus solution (To be implemented by Jan). We keep names of all files. We check where this function is used and remove the pWM part of the prefix - on all occurrences. Make sure that all the underscores are correct. Then we can merge.